patches/nan/chore_remove_deprecated_accessorsignatures.patch

```diff
@@ -0,0 +1,45 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Thu, 2 Jun 2022 15:45:21 +0200
+Subject: chore: remove deprecated AccessorSignatures
+
+Removed in https://chromium-review.googlesource.com/c/v8/v8/+/3654096
+This should be upstreamed to nan.
```

Comment:
Probably should include a link to https://github.com/nodejs/nan/pull/941 for visibility